src/caf/base/data_structures.py

```diff
@@ -342,8 +343,8 @@ def data(self, value: pd.Series | pd.DataFrame):
             raise TypeError(
                 "data must be a pandas DataFrame or Series. Input " f"value is {value.type}."
             )
-        if isinstance(value, pd.Series):
-            value = value.to_frame()
+        # if isinstance(value, pd.Series):
+        #     value = value.to_frame()
```

Comment:
Why is this commented out, doesn't `_dataframe_to_dvec` require a dataframe? If it can handle Series please can you update the type annotation for it.